mailing-list/main.js

```diff
@@ -170,6 +177,25 @@ async function main() {
     }
     console.log(`Found ${documents.length} documents`);
 
+    let threads = {};
+    for (let i = 0; i < documents.length; i++) {
+        const document = documents[i];
+        if (!threads[document.title]) {
+            threads[document.title] = {
+                id: document.id,
+                date: findEarliestTimestamp(documents, document.title),
+                url: document.url,
+            };
+        }
+
+        if (threads[document.title].id === document.id) {
+            document.type = "thread";
```

Comment:
I'd prefer "original_post" instead of "thread"

Reply:
Done